QuantLib.vcxproj

```diff
@@ -567,7 +567,7 @@
     <ClInclude Include="ql\experimental\barrieroption\vannavolgabarrierengine.hpp" />
     <ClInclude Include="ql\experimental\barrieroption\vannavolgadoublebarrierengine.hpp" />
     <ClInclude Include="ql\experimental\barrieroption\vannavolgainterpolation.hpp" />
-    <ClInclude Include="ql\experimental\barrieroption\wulinyongdoublebarrierengine.hpp" />
+    <ClInclude Include="ql\experimental\barrieroption\SuoWangDoubleBarrierEngine.hpp" />

```

Comment:
You changed the file name here (and in the other projects), but didn't actually rename the file so it won't be found by the compiler.